api/app/routers/fba_calc.py

```diff
@@ -94,16 +97,37 @@ def process_request(
 
     # extract variable from wf1 that we need to calculate the fire behaviour advisory.
     bui = raw_daily.get('buildUpIndex', None)
-    ffmc = raw_daily.get('fineFuelMoistureCode', None)
-    isi = raw_daily.get('initialSpreadIndex', None)
-    wind_speed = raw_daily.get('windSpeed', None)
+    temperature = raw_daily.get('temperature', None)
+    relative_humidity = raw_daily.get('relativeHumidity', None)
+    precipitation = raw_daily.get('precipitation', None)
+    # if user has specified wind speed as part of StationRequest, will need to
+    # re-calculate FFMC & ISI with modified value of wind speed
+    if requested_station.wind_speed is not None:
```

Comment:
Flip this around please - "if requested_station.wind_speed is None" reads easier.